### ISSUE TITLE
Styling Knowledgebase

### ISSUE THREAD

**Issue by GralfR** [cbdevnet/midimonster#64]
Opening the [Knowledgebase](https://kb.midimonster.net/) with Chrome (checked with version 58.0.3029.110) displays white blocks on white background, no text, even if I select anything.
With IE9 it's readable.

**Comment by cbdevnet** [comment]
Hmm this is strange, with Chrome on Linux (`Version 80.0.3987.162 (Developer Build)`) I see the content as it should be.

Will look into this on Windows, thanks for the report!

**Comment by julipatrusheva** [comment]
I just checked Chrome 58.x on Windows 10. I see the content as it should be as well.

@GralfR could you please check the website in incognito window?

**Comment by GralfR** [comment]
Incognito-window does not help. Same problem.
I also tried Firefox 40. This is okay.
With Avast-Browser the same problem as with chrome, because Avast is based on chrome.
Also hardware-acceleration on or off makes no difference. Adblocker deactivated.
The IE9 in document-mode IE9 shows text but no capital letters.

Your KB is the only page with this problem on my system (Win7 x64). Reboot did not help either.

**Comment by cbdevnet** [comment]
The Knowledgebase is currently using the sphinx-rtd (readthedocs.org) theme with very minor modifications, another site using this theme is the [theme's example page](https://sphinx-rtd-theme.readthedocs.io/en/stable/), could you tell us whether that renders okay for you?

**Comment by GralfR** [comment]
yes, the example is rendered correctly with black text and blue links on white background.
The KB has white links and white text. Only the headlines have black text on white background.

**Comment by cbdevnet** [comment]
Are you sure that you have Version 58? It seems to have been released in 2017, which in "browser years" is really old.

This makes the issue a bit hard to test for, since all Chrome installers include an auto-updater now that immediately wants to update to a later version. I'll try to fix some standards-compliance things I'm getting when running the site through a validator and I'll pull the latest version of the template over the next few days, I'll update here when it's done.

**Comment by GralfR** [comment]
Yes, I use an old chrome, but that's not the point. The much older IE9 and even the historic Firefox 40 renders correctly. That's why I do not autoupdate. autoupdates are the real virus of the internet. New versions tend to refuse things that worked before the update.
My other browser, the avast secure browser has version 81.1.4222.139, which is a much more current one and based on chromium. This one has the same problem.
So, I guess, it's one of Your modifications, 'cause the example is okay.

**Comment by cbdevnet** [comment]
Definitely agree with you on the auto-update thing, don't like them much myself.

Good to know that a newer one shows the same problem, could you possibly try to open the "inspect" view in Chrome (right click in page, "Inspect") and see if in the "Network" tab, anything shows up as failed (not a 200 OK or 304 REDIRECT)?

**Comment by GralfR** [comment]
all 18 requests have status 200 okay. Tried several reloads. No errors under network.

**Comment by GralfR** [comment]
strange thing:
In chrome I saved the faulty page to the local harddrive and opened the local file in chrome.
Now, I can read the text. The inspector has several errors under network: all woff and ttf fonts failed. Maybe the fonts are the problem...?
